docs/pages/1 - Intro to Mill.md

```diff
@@ -701,9 +701,16 @@ mill mill.scalalib.Dependency/showUpdates
 mill mill.scalalib.Dependency/showUpdates --allowPreRelease true # also show pre-release versions
 ```
 
-## IntelliJ Support
+## IDE Support
 
-Mill supports IntelliJ by default. Use `mill mill.scalalib.GenIdea/idea` to
+Mill supports any IDE that is compatible with [BSP](https://build-server-protocol.github.io/), such as IntelliJ, VSCode
+or Emacs. Use `mill mill.BSP/install` to generate the BSP project config for your build.
+
+This also configures your IDE to allow easy navigate & code-completion within your build file itself.
```

Comment:
This is interesting. Does it rely on ammonite's BSP implementation ?

Reply:
No, Idea just pulls all the deps and then figures the rest out.

Reply:
Right, so it does works with intellij, because of how it's got some built-in support for worksheets (and rudimentary support for ammonite scripts). Therefore, the current implementation will be insufficient for metals to support navigation/code-completion. 

I'd appreciate if this line was worded differently to reflect this. Something like : 

```suggestion
It also enables Intellij to provide navigation & code-completion features within your build file itself.
```